index.bs

```diff
@@ -953,33 +954,42 @@ spec: webidl
     using <code>{{Promise}}.all</code>.
   </p>
   <pre class='highlight'>
-    &lt;script&gt;
-      Promise.all([navigator.permissions.query({name:'geolocation'}),
-                   navigator.permissions.query({name:'notifications'})])
-      .then(function(result) {
-        console.log('Geolocation permission state is ' + result[0].state);
-        console.log('Notifications permission state is ' + result[1].state);
-      });
-    &lt;/script&gt;
+Promise.all([
+    navigator.permissions.query({ name: "geolocation" }),
+    navigator.permissions.query({ name: "notifications" })
+  ])
+  .then(([{ state: geoState }, { state: notifState }]) => {
+    console.log("Geolocation permission state is:", geoState);
+    console.log("Notifications permission state is:", notifState);
+  });
   </pre>
   </div>
   <div class="example" id="example-deviceId">
   <p>
-    This example is checking whether the page has the {{"camera"}}
-    permission for the first device in the list. Error checking omitted.
+    This example is checking the permission of the the first camera.
   </p>
   <pre class='highlight'>
-    &lt;script&gt;
-      navigator.mediaDevices.enumerateDevices()
-      .then(function(devices) {
-         return navigator.permissions.query({name:'camera',
-                                             deviceId: devices[0].deviceId})
-      })
-      .then(function(result) {
-         console.log('Camera permission to first camera is ' + result.state);
-      }
-      .catch(err =&gt; console.log('Bad things happened: ' + err.name));
-    &lt;/script&gt;
+navigator.mediaDevices
+  .enumerateDevices()
+  // filter out just video inputs
+  .then(
+    devices => devices.filter(({ kind }) => kind === "videoinput")
+  )
+  // get the deviceId of first camera, create query object
+  .then(
+    ([{ deviceId }]) => ({ name: "camera", deviceId })
+  )
+  // perform query
+  .then(
+    queryObj => navigator.permissions.query(queryObj)
+  )
+  // log the state
+  .then(
+    ({ state }) => console.log("Permission state of first camera is", state)
+  )
+  .catch(
+    (err) => console.error(err.stack)
```

Comment:
You don't need the ()s around `err` here. Do we want to be consistent about being completely minimal, or do we not care?